teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>In the above bar chart we compared</p><p>-&gt; In the above bar chart, we compared</p><p> </p><p>... four example <strong>drug</strong> molecules ...</p><p>The four examples are note necessarily drugs or are they? Maybe just delete?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>